Hybrid/src/main/java/com/microsoft/azure/management/storage/samples/ManageStorageAccount.java

```diff
@@ -0,0 +1,246 @@
+        final String storageAccountName = Utils.createRandomName("sa");
+        final String storageAccountName2 = Utils.createRandomName("sa2");
+        final String rgName = Utils.createRandomName("rgSTMS");
+        try {
+            // ===========================================================
+            // Create a resource group
+            ResourceGroup resourceGroup = azure.resourceGroups().define(rgName).withExistingSubscription()
+                    .withLocation(location).create();
+            System.out.println("Created a resource group with name: " + resourceGroup.name());
+
+            // ============================================================
+            // Create a storage account
+
+            System.out.println("Creating a Storage Account");
+            StorageAccount storageAccount = azure.storageAccounts().define(storageAccountName).withRegion(location)
+                    .withExistingResourceGroup(rgName).withKind(Kind.STORAGE)
+                    .withSku(new Sku().withName(SkuName.STANDARD_LRS)).create();
+
+            System.out.println("Created a Storage Account:");
+            Utils.print(storageAccount);
+
+            // ============================================================
+            // Get | regenerate storage account access keys
+
+            System.out.println("Getting storage account access keys");
+            List<StorageAccountKey> storageAccountKeys = azure.storageAccounts().inner()
+                    .listKeys(rgName, storageAccountName).keys();
+
+            Utils.print(storageAccountKeys);
+
+            System.out.println("Regenerating first storage account access key");
+
+            storageAccountKeys = azure.storageAccounts().inner()
+                    .regenerateKey(rgName, storageAccountName, storageAccountKeys.get(0).keyName()).keys();
+
+            Utils.print(storageAccountKeys);
+
+            // ============================================================
+            // Create another storage account
+
+            System.out.println("Creating a 2nd Storage Account");
+
+            StorageAccount storageAccount2 = azure.storageAccounts().define(storageAccountName2).withRegion(location)
+                    .withExistingResourceGroup(rgName).withKind(Kind.STORAGE)
+                    .withSku(new Sku().withName(SkuName.STANDARD_LRS)).create();
+
+            System.out.println("Created a Storage Account:");
+            Utils.print(storageAccount2);
+
+            // ============================================================
+            // Update storage account by enabling encryption
+
+            System.out.println("Enabling blob encryption for the storage account: " + storageAccount2.name());
+            storageAccount2.update()
+                    .withEncryption(
+                            new Encryption().withServices(new EncryptionServices().withBlob(new EncryptionService())))
+                    .apply();
```

Comment:
if we can extract param out, then this could be more readable 

```java
Encryption encryptSettings = new Encryption()
        .withServices(new EncryptionServices()
                .withBlob(new EncryptionService()));
        
storageAccount2.update()
        .withEncryption(encryptSettings)
        .apply();
```